<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>590</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>新宋体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>291</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelTitle">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>车辆管理</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>291</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageCar">
       <widget class="QSplitter" name="splitter">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>811</width>
          <height>471</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QToolBox" name="toolBox">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>546</width>
            <height>411</height>
           </rect>
          </property>
          <attribute name="label">
           <string>出售车辆</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget1">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>120</y>
             <width>251</width>
             <height>23</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>报价</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditPrice"/>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>万元</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget2">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>170</y>
             <width>181</width>
             <height>23</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>数量</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget4">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>280</y>
             <width>251</width>
             <height>23</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>金额</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditTotal"/>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>万元</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget5">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>330</y>
             <width>242</width>
             <height>30</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPushButton" name="buttonSure">
              <property name="text">
               <string>确定</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="buttonCancle">
              <property name="text">
               <string>取消</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget6">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>10</y>
             <width>222</width>
             <height>23</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>厂家</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxfactory"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget7">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>60</y>
             <width>222</width>
             <height>23</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>品牌</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_11">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxbrand"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QLabel" name="labelLast">
           <property name="geometry">
            <rect>
             <x>187</x>
             <y>231</y>
             <width>16</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>20</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>41</x>
             <y>231</y>
             <width>64</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>剩余数量</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>546</width>
            <height>411</height>
           </rect>
          </property>
          <attribute name="label">
           <string>新车入库</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0" colspan="5">
            <widget class="QTableView" name="tableView"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>id</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2" colspan="2">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>factory</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>name</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="lineID">
             <property name="text">
              <string>9</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="3">
            <widget class="QLineEdit" name="lineFactory">
             <property name="text">
              <string>AA</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLineEdit" name="lineName">
             <property name="text">
              <string>AAB</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>price</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>num</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="3">
            <widget class="QLineEdit" name="linePrice">
             <property name="text">
              <string>25</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="3" colspan="2">
            <widget class="QLineEdit" name="lineNum">
             <property name="text">
              <string>10</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="5">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="buttonAdd">
               <property name="text">
                <string>添加</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonDel">
               <property name="text">
                <string>删除</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonSub">
               <property name="text">
                <string>提交</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonRev">
               <property name="text">
                <string>撤销</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QTextEdit" name="textEdit">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="pageOther">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="4">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>15</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_8"/>
        </item>
        <item row="1" column="1" colspan="6">
         <layout class="QHBoxLayout" name="horizontalLayout_9"/>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>250</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>厂家</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" colspan="2">
         <widget class="QComboBox" name="comboBoxOtherFactory"/>
        </item>
        <item row="2" column="5">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>品牌</string>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QComboBox" name="comboBoxOtherBrand"/>
        </item>
        <item row="2" column="7">
         <spacer name="horizontalSpacer_12">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>259</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="3" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="4" column="0" colspan="4">
         <widget class="QChartView" name="barCharts"/>
        </item>
        <item row="4" column="5" colspan="3">
         <widget class="QChartView" name="pieCharts"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>销售管理</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>车辆管理</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>销售统计</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
